mep/accounts/admin.py

```diff
@@ -278,11 +278,12 @@ class BorrowAdminForm(forms.ModelForm):
 
     class Meta:
         model = Borrow
-        fields = ('account', 'item', 'partial_start_date', 'partial_end_date',
-            'notes')
+        fields = ('account', 'item', 'item_status', 'partial_start_date',
+            'partial_end_date', 'notes')
         widgets = {
             'account': AUTOCOMPLETE['account'],
-            'item': AUTOCOMPLETE['item']
+            'item': AUTOCOMPLETE['item'],
+            'item_status' : forms.RadioSelect
```

Comment:
@thatbudakguy I meant to remove this before I checked my code in - that's the only reason I'm using a separate form class for the list edit.  There's something to be said for consistency, though - should we use the same widget for both?  If so, I think the display tweaks you added for the list view need some adjustment for the single item edit form.  

If we go that route, we should consolidate to a single form.  I'd also be ok with just deleting this line and switching it back to a drop-down for the single item edit.